ipapython/certdb.py

```diff
@@ -83,7 +83,8 @@ class NSSDatabase(object):
     # got too tied to IPA server details, killing reusability.
     # BaseCertDB is a class that knows nothing about IPA.
     # Generic NSS DB code should be moved here.
-    def __init__(self, nssdir=None):
+    def __init__(self, nssdir=None, password_filename=None):
+        self.password_filename = password_filename
```

Comment:
We might but password file does not need to exist in general. I think this assumption is made in CertDB instead.
I can add it nonetheless.